src/controller/CHIPPersistentStorageDelegate.h

```diff
@@ -79,6 +79,19 @@ class DLL_EXPORT PersistentStorageDelegate
      */
     virtual void GetKeyValue(const char * key) = 0;
 
+    /**
+     * @brief
+     *   Lookup the key and call delegate object with it's stringified value.
```

Comment:
There's no delegate-calling here, afaict.  We're just returning the value in the outparam?